src/Fantomas/SourceParser.fs

```diff
@@ -1403,4 +1403,9 @@ let rec (|UppercaseSynExpr|LowercaseSynExpr|) (synExpr:SynExpr) =
 let rec isEmptySynSimplePats (ssp:SynSimplePats) =
```

Comment:
I noticed this function is not being used anywhere. I know this has nothing to do with your PR but would you mind removing it?